fe/fe-core/src/test/java/com/starrocks/sql/plan/AggregateTest.java

```diff
@@ -819,6 +820,16 @@ public void testWindowFunnel() throws Exception {
         plan = getFragmentPlan(sql);
         assertContains(plan, "window_funnel(1800, 11: L_SHIPDATE, 3, 18: expr)");
 
+        sql =
+                "select L_ORDERKEY,window_funnel(-1, L_SHIPDATE, 3, [L_PARTKEY = 1]) from lineitem_partition_colocate group by L_ORDERKEY;";
+	try {
+            plan = getFragmentPlan(sql);
+        } catch (SemanticException e) {
```

Comment:
could use ExpectedException, refer to ColocateTableTest

Reply:
> could use ExpectedException, refer to ColocateTableTest

done